scoring_engine/checks/ssh.py

```diff
@@ -3,7 +3,7 @@
 
 class SSHCheck(BasicCheck):
     required_properties = ['commands']
-    CMD = CHECKS_BIN_PATH + '/ssh_check {0} {1} {2} {3} {4}'
+    CMD = CHECKS_BIN_PATH + "/ssh_check '{0}' '{1}' '{2}' '{3}' {4}"
```

Comment:
this method was not enough to deal with arbitrary characters inside passwords, a better sanitization/passing method will be needed